app/controllers/admin/schedules_controller.rb

```diff
@@ -31,14 +39,17 @@ def update
         error_message = "Could not find event GUID: #{params[:event]}"
```

Comment:
Both event and room use the guid, so I think it is clearer in the view to call it event, to avoid confusion

https://github.com/openSUSE/osem/pull/1094/files#diff-4db956c0d81325278cf026c6f02ae666R122